### PR TITLE
Add coffea.casa executor

### PR DESCRIPTION
**Pull request by oshadura**
Based on https://pocketcoffea.readthedocs.io/en/main/running.html#adding-support-for-a-new-executor-site

### CODE REVIEW

**Comment by valsdav** [comment]
Hi @oshadura thanks a lot!  
Yesterday we moved the scripts inside `pocket_coffea/scripts` to implement a CLI based on click. 
Can you please fix the conflict? :)

File: pocket_coffea/executors/executors_casa.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    def __init__(self, run_options, outputdir, **kwargs):
+        self.outputdir = outputdir
+        super().__init__(run_options, **kwargs)
+        
+    def setup(self):
+        ''' Start the DASK cluster here'''
+        # At coffea-casa we have preconfigured Dask HTCondor cluster for you, please just use it available at tls://localhost:8786
+        from distributed import Client
+        self.dask_client = Client("tls://localhost:8786")
```
Comment (valsdav) [inline pocket_coffea/executors/executors_casa.py:16]: If this port always the same? Is it needed to put it in the default parameters?

File: pocket_coffea/executors/executors_casa.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        # Install pocket_coffea to dask workers via Dask scheduler plugin PipInstall
+        # PLEASE CHANGE if you are using fork or different tag / branch
+        from dask.distributed import PipInstall
+        plugin = PipInstall(packages=["git+https://github.com/PocketCoffea/PocketCoffea.git@main"])
```
Comment (valsdav) [inline pocket_coffea/executors/executors_casa.py:20]: Can be configured later

File: pocket_coffea/executors/executors_casa.py (hunk 1 above)
Comment (oshadura) [reply pocket_coffea/executors/executors_casa.py:16]: It is always the same for each user

**Pull request merged**